brain/compartmentReportView.cpp

```diff
@@ -95,45 +95,18 @@ std::future<CompartmentReportFrame> CompartmentReportView::load(
     return _impl->readerImpl->threadPool.post(loadFrameTask);
```

Comment:
Wasn't it the idea to get rid of CompartmentReportFrame completely and use a simple double floatsPtr struct instead (as Frames for multiple vectors)? The current API in CompartmentReportView looks a bit asymmetrical otherwise. Here you can use a deferred async to transfer the output from the brion::CompartmentReport::loadFrame to the struct.